ddtrace/contrib/django/__init__.py

```diff
@@ -137,9 +150,13 @@
 3. Remove ``TraceMiddleware`` or ``TraceExceptionMiddleware`` if used in
    ``settings.py``.
 
-3. Enable Django tracing automatically via `ddtrace-run`` or manually by
+4. Enable Django tracing automatically via `ddtrace-run`` or manually by
    adding ``ddtrace.patch_all()`` to ``settings.py``.
 
+5. Set environment variable ``DD_DJANGO_USE_LEGACY_RESOURCE_FORMAT=true`` to
```

Comment:
Would also be good to mention `config. use_legacy_resource_format` for those configuring in code.